src/jit/importer.cpp

```diff
@@ -1860,18 +1860,17 @@ GenTreePtr Compiler::impMethodPointer(CORINFO_RESOLVED_TOKEN* pResolvedToken, CO
 // getRuntimeContextTree: find pointer to context for runtime lookup.
 //
 // Arguments:
-//    pLookup - how to do lookup.
+//    kind - lookup kind.
 //
 // Return Value:
 //    Return GenTree pointer to generic shared context.
 //
 // Notes:
 //    Reports about generic context using.
 
-GenTreePtr Compiler::getRuntimeContextTree(CORINFO_LOOKUP* pLookup)
+GenTreePtr Compiler::getRuntimeContextTree(CORINFO_RUNTIME_LOOKUP_KIND kind)
 {
     GenTreePtr                  ctxTree = nullptr;
```

Comment:
The code formatting enforcement insist on having just a single space between `GenTreePtr` and `ctxTree` after your change.